src/main/kotlin/com/sourcegraph/cody/agent/protocol/ProtocolCommand.kt

```diff
@@ -0,0 +1,9 @@
+package com.sourcegraph.cody.agent.protocol
+
+data class ProtocolCommand(
+    val title: TitleParams? = null,
+    val command: String? = null,
+    val tooltip: String? = null,
+    // First argument is always task ID (string).
```

Comment:
Now that we're not using commands for accept, cancel, undo—what do we need this for?

If we do need it, this is not true for many commands. Let's elaborate the comment a bit, maybe this is a TODO.

Reply:
If it would be the case it's still needed, and it is required for some commands but not for the others, why do not make it separate param only for commands which need it?

Reply:
I've ripped out the entire code path; it's easy enough to put back in if we ever need it.